doc/users/plotting/colormaps/lightness.py

```diff
@@ -79,10 +61,22 @@
 
             # Plot colormap L values
             # Do separately for each category so each plot can be pretty
-            # to make scatter markers change color along plot: http://stackoverflow.com/questions/8202605/matplotlib-scatterplot-colour-as-a-function-of-a-third-variable
-            if cmap_category=='Sequential':
+            # to make scatter markers change color along plot:
+            # http://stackoverflow.com/questions/8202605/matplotlib-scatterplot-colour-as-a-function-of-a-third-variable
+            if cmap_category=='Perceptually Uniform Sequential':
```

Comment:
Not these since they are just plots embedded in the prose text. We could do this. They may not be pep8 valid after this pr. But they are much better than before